terraform/go.mod

```diff
@@ -9,24 +9,20 @@ replace (
 )
 
 require (
-	github.com/gravitational/configure v0.0.0-20180808141939-c3428bd84c23 // indirect
-	github.com/gravitational/teleport v1.3.3-0.20210121233412-127693c315fb
+	github.com/gogo/protobuf v1.3.1
+	github.com/gravitational/teleport v1.3.3-0.20210205231255-9a469a32369d
```

Comment:
`github.com/gravitational/teleport/api` is now a self contained go pkg, so you shouldn't need to import the full teleport package. Since the current import you are using is relatively old, it will have some differences. Message me if you run into any issues here.

Reply:
Done, no issues here, thanks.